Dockerfile.base

```diff
@@ -112,27 +124,4 @@ RUN mkdir /root/.ssh/ && ssh-keyscan github.com | tee -a /root/.ssh/known_hosts
 
 RUN echo "session required pam_limits.so" | tee --append /etc/pam.d/common-session > /dev/null
 
-
-FROM ssh-configuration as development-dependencies
-
-RUN apt-get update && apt-get install -y \
-    clang \
-    gdb \
-    python3 \
-    python3-dev \
-    python3-pip \
-    tar \
-    unzip \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# install python requirements
-RUN pip3 install numpy setuptools pybind11
-
-# install google dependencies
-COPY --from=google-dependencies /usr/include/gtest /usr/include/gtest
-COPY --from=ghcr.io/epfl-lasa/control-libraries/proto-dependencies /usr/local/include/google /usr/local/include/google
```

Comment:
Why don't we have to do that anymore?

Reply:
Hmm good point, I think I took it out while testing because `--from=ghcr.io/epfl-lasa/control-libraries/proto-dependencies` did not yet work without the multiarch build, but forgot to put it back. The gtest deps are still there so it was building fine, but the protocol would not have built.